packages/lpa-questionnaire-web-app/tests/unit/middleware/clear-uploaded-files.test.js

```diff
@@ -0,0 +1,31 @@
+const clearUploadedFilesMiddleware = require('../../../src/middleware/clear-uploaded-files');
+const { mockReq, mockRes } = require('../mocks');
+
+describe('middleware/clear-uploaded-files', () => {
+  let req;
+  let next;
+
+  beforeEach(() => {
+    req = mockReq();
+    next = jest.fn();
+
+    jest.resetAllMocks();
```

Comment:
If you're resetting req and next on lines 9 and 10, do you need to reset all mocks?